tests/common/types.py

```diff
@@ -98,7 +98,7 @@ class Prop(Column):
     def __init__(self, index: int, name: str, ptype: str):
         super().__init__(index)
         self._name = name
-        if ptype not in ['string', 'int', 'double']:
+        if ptype not in ['string', 'int', 'double', 'bool']:
```

Comment:
So, there is no time props?

Reply:
Or any complex props, like List, Map?

Reply:
Yes, the generator doesn't support temporal types, you could submit an issue. Nebula doesn't support list/map as property.

Reply:
> Yes, the generator doesn't support temporal types, you could submit an issue. Nebula doesn't support list/map as property.

This is not what we required previously, this need further refinement in the fufure.